server/src/main/java/com/cloud/vm/UserVmManager.java

```diff
@@ -55,6 +55,10 @@
 
     static final int MAX_USER_DATA_LENGTH_BYTES = 2048;
 
+    public static enum UserVmType {
```

Comment:
@Pearl1594  there is a common enum "VirtualMachine.Type" where all the VM types are defined, can use the same?